classes/Link.php

```diff
@@ -743,21 +743,14 @@ public function getAdminLink($controller, $withToken = true, $sfRouteParams = ar
                     'AdminModulesManage' => 'admin_module_manage',
                     'AdminModulesNotifications' => 'admin_module_notification',
                     'AdminModulesSf' => 'admin_module_manage',
+                    'AdminCustomerPreferences' => 'admin_customer_preferences',
```

Comment:
Duplicate line 737

Reply:
not anymore :)

Reply:
Line was sorted and you broke it :crying_cat_face: